### PR TITLE
8/4: Add PerspectiveDecision Class

### PR DESCRIPTION
**Pull request by annemariecabs**
Add the PerspectiveDecision class (a container for the decision returned by Perspective).

### CODE REVIEW

File: backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  /**
+   * Returns whether there is an appropriate story stored in this object.
+   *
+   * @return if there was a story deemed appropriate by analysis engine.
+   */
+  public boolean hasStory() {
+    return hasStory;
+  }
```
Comment (ykjchen) [inline backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java:52]: If you decide to go with the exception, it sounds like this property would no longer be needed.

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java]: Removed this property.

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java]: Done

File: backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  /**
+   * Constructs a PerspectiveDecision object from a String story.
+   *
+   * @param story A story deemed appropriate by the analysis engine.
+   * @throws IllegalArgumentException if story is null
+   */
+  public PerspectiveDecision(String story) throws IllegalArgumentException {
+    if (story == null)
+      throw new IllegalArgumentException("story should never be null");
```
Comment (ykjchen) [inline backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java:30]: Google style requires braces even where optional.

Comment (UdayKalra) [inline backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java:30]: Nit: Capitalize Story for proper sentence thrown with exception.

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java:30]: Will do. Thanks!

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java:30]: Will add. Thanks!

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java]: Done

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveDecision.java]: Done

**Pull request merged**